test/e2e/e2e_test.go

```diff
@@ -12,20 +12,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/pointer"
 
-	autoscalingv1 "github.com/openshift/cluster-resource-override-admission-operator/pkg/apis/autoscaling/v1"
-	"github.com/openshift/cluster-resource-override-admission-operator/test/helper"
+	appsv1 "github.com/openshift/run-once-duration-override-operator/pkg/apis/apps/v1"
+	"github.com/openshift/run-once-duration-override-operator/test/helper"
 )
 
 func TestClusterResourceOverrideAdmissionWithOptIn(t *testing.T) {
```

Comment:
None of these tests are working currently, I'm planning to make all of them working in the next commit.